scalardl/project.clj

```diff
@@ -10,12 +10,12 @@
                  [cc.qbits/hayt "4.1.0"]]
   :profiles {:dev {:dependencies [[tortue/spy "2.0.0"]]
                    :plugins [[lein-cloverage "1.1.2"]]}
-             :use-released {:dependencies [[com.scalar-labs/scalardl-java-client-sdk "3.0.1" :exclusions [org.slf4j/slf4j-log4j12]]]}
+             :use-released {:dependencies [[com.scalar-labs/scalardl-java-client-sdk "3.3.1" :exclusions [org.slf4j/slf4j-log4j12]]]}
              :use-jars {:dependencies [[org.bouncycastle/bcpkix-jdk15on "1.59"]
                                        [org.bouncycastle/bcprov-jdk15on "1.59"]
                                        [com.google.inject/guice "4.2.0"]
                                        [com.moandjiezana.toml/toml4j "0.7.2"]
-                                       [com.google.protobuf/protobuf-java-util "3.13.0"]
+                                       [com.google.protobuf/protobuf-java-util "3.19.2"]
```

Comment:
Do we really need this?
Why do we need to define `protobuf-java-util` here?

Reply:
In the daily/weekly Jepsen tests, we use the `:use-jars` profile and build the following jars and put them to the test server:
https://github.com/scalar-labs/scalar-jepsen/blob/ae5cd750f43ce684ca032b94ef8ffe9a1990888b/scalardl/project.clj#L21-L23

However, since the jars don't include their dependencies, we need to add them in the `:use-jars` profile. That's why we need the dependencies.

Reply:
> Do we really need this?

Yes, Without this change test will be failed
```
INFO [2022-01-19 07:40:06,822] jepsen node n3 - scalardl.cas register a certificate and contracts
WARN [2022-01-19 07:40:07,314] jepsen node n3 - io.grpc.stub.ClientCalls$ThreadlessExecutor Runnable threw exception
java.lang.NoSuchMethodError: com.google.protobuf.GeneratedMessageV3.isStringEmpty(Ljava/lang/Object;)Z
	at com.scalar.dl.rpc.CertificateRegistrationRequest.getSerializedSize(CertificateRegistrationRequest.java:240)
	at io.grpc.protobuf.lite.ProtoInputStream.available(ProtoInputStream.java:108)
	at io.grpc.internal.MessageFramer.getKnownLength(MessageFramer.java:205)
	at io.grpc.internal.MessageFramer.writePayload(MessageFramer.java:137)
	at io.grpc.internal.AbstractStream.writeMessage(AbstractStream.java:65)
	at io.grpc.internal.ForwardingClientStream.writeMessage(ForwardingClientStream.java:37)
	at io.grpc.internal.DelayedStream$6.run(DelayedStream.java:257)
	at io.grpc.internal.DelayedStream.drainPendingCalls(DelayedStream.java:163)
	at io.grpc.internal.DelayedStream.setStream(DelayedStream.java:132)
	at io.grpc.internal.DelayedClientTransport$PendingStream.createRealStream(DelayedClientTransport.java:358)
	at io.grpc.internal.DelayedClientTransport$PendingStream.access$300(DelayedClientTransport.java:341)
	at io.grpc.internal.DelayedClientTransport$5.run(DelayedClientTransport.java:300)
	at io.grpc.stub.ClientCalls$ThreadlessExecutor.waitAndDrain(ClientCalls.java:740)
```